### ISSUE TITLE
Error checking https://www.jonloomer.com/

### ISSUE THREAD

**Issue by garrettr** [mozilla-services/Heartbleed#1]
Our service returns an error: http://heartbleed.prod.mozaws.net/bleed/www.jonloomer.com

```
{"code":2,"data":"","error":"heartbleed: timeout"}
```

Original service returns "not affected": https://filippo.io/Heartbleed/#www.jonloomer.com

**Comment by oremj** [comment]
He's squashed quite a few bugs. I tried rebasing our fork, but there are a ton of merge conflicts around the dynamodb stuff. @jrconlin do you think it's worth it to merge in his master?

**Comment by jrconlin** [comment]
A timeout error occurs in the heartbleed.go file. It's triggered when the site takes longer than 6 seconds to respond.

https://github.com/mozilla-services/Heartbleed/blob/master/bleed/heartbleed.go#L120

The site took a bit to respond, but far less than 6 seconds.

I'm guessing that it might be something he's fixed up. I'll merge to the latest and see if it's still a problem.

**Comment by oremj** [comment]
Yeah, I noticed there was a commit, in the last few days, specifically raising timeouts across the board.

**Comment by garrettr** [comment]
Best course of action seems to be merging their latest with our changes. @jrconlin Do you think you could do that today? (we're trying to launch the add-on in the afternoon)

**Comment by garrettr** [comment]
I'm getting lots of other bug reports with our check server today (false positives and negatives). For example, addons.mozilla.org is "vulnerable": http://heartbleed.prod.mozaws.net/bleed/addons.mozilla.org

**Comment by jrconlin** [comment]
I've updated my dev branch to include the latest heartbleed changes:
https://github.com/jrconlin/Heartbleed/tree/moz

Since this does not appear to be a normal fork, nor am I a contributor, I can't generate a PR.

I've tested this against www.cloudflarechallenge.com, jonloomer.com, and example.com and gotten expected results for all. 

I've also broken out the mozilla changes to their own file: bleed_serve_moz.go

I plan on putting the cache bits into their own file and pushing changes back upstream.
